core/install.js

```diff
@@ -12,20 +12,35 @@
 var npm = require('npm'),
     deasync = require('deasync'),
     fs = require('fs'),
-    load = deasync(npm.load);
+    load = deasync(npm.load),
+    csHasLoaded = false;
 
 load({loglevel: 'silent'});
 var inst = deasync(npm.commands.install),
-upd = deasync(npm.commands.update),
+    upd = deasync(npm.commands.update),
 
-install = function(name) {
-    console.info('Installing "' + name + '" from npm.');
-    inst([name]);
-    console.info('Installation complete.');
-};
+    // inject require modifications into all coffeescript code because babel wont
+    coffeescriptRequireInjector = function () {
+        if (!csHasLoaded && global.coffeescriptLoaded) {
+            var cs = require("coffee-script"),
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/theprojectshop/Kassy/file/2737028527/issues/source?bid=3282987&fileBranchId=3347522#l25)